kafka-ui-e2e-checks/src/main/java/com/provectus/kafka/ui/pages/topic/TopicCreateEditForm.java

```diff
@@ -93,9 +104,9 @@ public TopicCreateEditForm selectRetentionBytes(Long optionValue) {
     }
 
     @Step
-    public TopicDetails sendData() {
-        clickByJavaScript($x("//button[@type='submit']"));
-        return new TopicDetails();
+    public TopicCreateEditForm submitCreateTopicBtn() {
```

Comment:
this method does click on button. why 'submit'??

Reply:
Renamed